tx.go

```diff
@@ -672,27 +673,27 @@ func (tx *Tx) Page(id int) (*PageInfo, error) {
 // TxStats represents statistics about the actions performed by the transaction.
 type TxStats struct {
 	// Page statistics.
-	PageCount int // number of page allocations
-	PageAlloc int // total bytes allocated
+	PageCount int64 // number of page allocations
```

Comment:
I would go a step forward: 

1. Warn on the tx.Stats() method that direct usage of the fields is deprecated
due to lack of safety on e.g. 32-bit architectures. 

2. Add to this class 2 methods for each field:

func (t* TxStats)IncrementPageCount(delta int64) {
 return atomic...
}

func (t* TxStats)GetPageCount(delta int64) {
 return atomic.LoadInt64(...)
}

such that user's can write safe code without depending on the arch-properties.

Reply:
Great point!

The only minor comment is about the name on the Getter method, It isn't a best practice to add the prefix `Get`, for example, `PageCount` is better than `GetPageCount` in above example. FYI. https://go.dev/doc/effective_go#Getters

Reply:
Agree.  But we already have `txStats.PageCount` field. 
I assume we cannot rename it (within major version) to `txStats.pageCount` as it would be breaking change of public API surface. 
We also cannot have a field and method with the same name in a single class.

Reply:
I would suggest to add comment to notify that we are planning not to expose each field, e.g change `PageCount` to `pageCount` .